core/shared/src/main/scala/scalaz/effect/package.scala

```diff
@@ -2,7 +2,15 @@
 
 package scalaz
 
+import scala.AnyRef
+
 package object effect {
+  type Error            = java.lang.Error
+  type Exception        = java.lang.Exception
+  type InternalError    = java.lang.InternalError
+  type OutOfMemoryError = java.lang.OutOfMemoryError
+  type RuntimeException = java.lang.RuntimeException
+  type Throwable        = java.lang.Throwable
```

Comment:
I'm not sure we want to expose these... ideally not.

Reply:
I just ported that 1:1 from sz8. You want me to remove it?

Reply:
I removed it. It was introduced due to the no-imports thing but we don't need it anyway.